MacaoProject/Macao/Card A.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Macao
+{
+    public class Card_A
```

Comment:
the name of the class is not correct. read about class naming convention https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names